src/cli/help.js

```diff
@@ -7,6 +7,7 @@ import dedent from "../util/dedent";
 import {help as loadHelp} from "./load";
 import {help as analyzeHelp} from "./analyze";
 import {help as pagerankHelp} from "./pagerank";
```

Comment:
`pagerankHelp` is somehow missing below and shows in the linter as unused import.

Reply:
Fixed.